### PR TITLE
Prepare for adding more test cases

### PR DESCRIPTION
**Pull request by angelikatyborska**
Before we deal with the open issues, it would be good to have a way to add more tests for corner cases instead of adding everything to the single example in `test_data`.

### CODE REVIEW

File: lib/representer.ex [hunk 1]
```diff
@@ -6,7 +6,7 @@ defmodule Representer do
   def process(file, code_output, mapping_output) do
     {represented_ast, mapping} = represent(file)
 
-    File.write!(code_output, Macro.to_string(represented_ast))
+    File.write!(code_output, Macro.to_string(represented_ast) <> "\n")
```
Comment (angelikatyborska) [inline lib/representer.ex:9]: My IDE is configured to auto-add trailing new lines to all files that I save. It's a good practice so that the files can be nicely read in a terminal without the prompt being printed on the same line as the last line of the file.

To be able to keep my settings and work in the IDE, I would need to add this tiny innocent new line to the produced representation 😇

**Comment by angelikatyborska** [comment]
I went for it and made it less DRY so that we can run a single test by specifying the LOC `mix test test/representer_test.exs:9`

**Review by neenjaw — APPROVED** [review]
